shell/app-shell/elements/arc-host.js

```diff
@@ -111,7 +111,8 @@ class ArcHost extends Xen.Debug(Xen.Base, log) {
     // make an id
     const id = 'app-shell-' + ArcsUtils.randomId();
     // construct storageKey
-    const storageKey = config.useStorage ? `${Firebase.storageKey}/arcs/${key}` : null;
+    const storageKey = config.useStorage ?
+          config.storageKeyBase || `${Firebase.storageKey}/arcs/${key}` : null;
```

Comment:
What about `/arcs/{$key}` part of the key?

Reply:
I suspect the line should be `const storageKey =config.storageKey;` and let `config` do the conditional, but we can do that later if it's inconvenient.

Reply:
For pouch I'll set this manually for the moment.

Reply:
I am just curious, what is problematic about including the `/arcs/${key}` now? I am pretty sure we'll need this to use Couch for M1 demos so we'd be looking at revising this within a matter of days in any case I would think?

Reply:
Okay, this was a misunderstanding on my part. I had assumed that this data was more global in scope than it really is.